specifications/api/swagger.yaml

```diff
@@ -1022,6 +1053,19 @@ paths:
           description: An optional filter on the address state.
       responses: *responsesListAddresses
 
+  /externalTransactions:
```

Comment:
Who decided the endpoint name? This is camelCase but there is another endpoint in kebab-case. I prefer the latter for URLs.

Reply:
so change to `external-transactions` or maybe `transactions/external` ?

Reply:
changed to `external-transactions`